sxt/base/curve/example_element.h

```diff
@@ -0,0 +1,96 @@
+/** Proofs GPU - Space and Time's cryptographic proof algorithms on the CPU and GPU.
+ *
+ * Copyright 2023-present Space and Time Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cstdint>
+#include <iostream>
+
+#include "sxt/base/macro/cuda_callable.h"
+
+namespace sxt::bascrv {
+//--------------------------------------------------------------------------------------------------
+// element97
+//--------------------------------------------------------------------------------------------------
+struct element97 {
+  uint32_t value;
+  bool marked = false;
+
+  element97() noexcept = default;
+
+  constexpr element97(uint32_t val) noexcept : value{val % 97u} {}
+
+  static constexpr element97 identity() noexcept { return {0}; }
+
+  bool operator==(const element97&) const noexcept = default;
+};
```

Comment:
From the code, I assume `97` was chosen arbitrarily for this example element. Is that correct? If I am incorrect, maybe a comment would help here?

Reply:
Added documentation. Thank you.